src/app/src/pages/evals/components/EvalsDataGrid.tsx

```diff
@@ -338,126 +413,143 @@ export default function EvalsDataGrid({
+              transition: 'background-color 0.2s ease',
+              '&:hover': {
+                backgroundColor: 'action.hover',
+              },
+            },
+            '& .MuiDataGrid-row--disabled': {
+              cursor: 'default',
+              opacity: 0.7,
+              pointerEvents: 'none',
+            },
+            '& .focused-row': {
               backgroundColor: 'action.selected',
+              cursor: 'default',
+              '&:hover': {
+                backgroundColor: 'action.selected',
+              },
             },
-          },
-          '& .MuiDataGrid-cell': {
-            borderColor: 'divider',
-          },
-          '& .MuiDataGrid-columnHeaders': {
-            backgroundColor: 'background.default',
-            borderColor: 'divider',
-          },
-          '& .MuiDataGrid-selectedRow': {
-            backgroundColor: 'action.selected',
-          },
-          '--DataGrid-overlayHeight': '300px',
-        }}
-        onRowSelectionModelChange={setRowSelectionModel}
-        rowSelectionModel={rowSelectionModel}
-        initialState={{
-          sorting: {
-            sortModel: [{ field: 'createdAt', sort: 'desc' }],
-          },
-          pagination: {
-            paginationModel: { pageSize: 50 },
-          },
-        }}
-        pageSizeOptions={[10, 25, 50, 100]}
-        isRowSelectable={(params) => params.id !== focusedEvalId}
-      />
-    </Paper>
+            '& .MuiDataGrid-cell': {
+              borderColor: 'divider',
+            },
+            '& .MuiDataGrid-columnHeaders': {
+              backgroundColor: 'background.default',
+              borderColor: 'divider',
+            },
+            '& .MuiDataGrid-selectedRow': {
+              backgroundColor: 'action.selected',
+            },
+            '--DataGrid-overlayHeight': '300px',
+          }}
+          onRowSelectionModelChange={setRowSelectionModel}
+          rowSelectionModel={rowSelectionModel}
+          initialState={{
+            sorting: {
+              sortModel: [{ field: 'createdAt', sort: 'desc' }],
+            },
+            pagination: {
+              paginationModel: { pageSize: 50 },
+            },
+          }}
+          pageSizeOptions={[10, 25, 50, 100]}
+          isRowSelectable={(params) => params.id !== focusedEvalId}
+        />
+      </Paper>
+
+      {/* Delete confirmation dialog */}
+      <Dialog
+        open={confirmDeleteOpen}
+        onClose={handleCancelDelete}
+        maxWidth={false}
+        fullWidth={false}
+        slotProps={{ paper: { sx: { minWidth: 350 } } }}
+      >
+        <DialogTitle>
+          {`Delete ${rowSelectionModel.length} eval${rowSelectionModel.length === 1 ? '' : 's'}?`}
+        </DialogTitle>
+        <DialogContent>
+          <Typography variant="body2" color="text.secondary">
+            This action cannot be undone.
+          </Typography>
+        </DialogContent>
+        <DialogActions>
```

Comment:
please take one more pass at the styles on this component. The margin / padding is not symmetric .

Reply:
It looks worse with symmetric padding so I'm going to leave as is:

<img width="1594" height="865" alt="Image" src="https://github.com/user-attachments/assets/c12cc801-cba8-4ede-8db1-e3d1632372fb" />